segmentation/utils/synthetic_spatial_datagen.py

```diff
@@ -7,35 +7,38 @@
 import xarray as xr
 import copy
 
+from random import seed
+from random import randint
 from segmentation.utils import spatial_analysis_utils as sau
+from segmentation.utils import visualize as viz
 from scipy.spatial.distance import cdist
 from skimage.measure import label
 
 
 def direct_init_dist_matrix(num_A=100, num_B=100, num_C=100,
-                            distr_AB={'mean': 100, 'var': 1}, distr_AC={'mean': 20, 'var': 1},
+                            distr_AB=(10, 1), distr_random=(200, 1),
```

Comment:
Now that we only have one function that actually makes a distance matrix, I think we can give this a more interpretable name. 

Something like create_synthetic_dist_matrix, generate_test_distance_matrix, etc etc.